lib/the_little_thinkers_space/file_compressor.ex

```diff
@@ -28,12 +28,15 @@ defmodule TheLittleThinkersSpace.FileCompressor do
     output_path = "#{path}_output#{filename}"
 
     with :ok <- File.rename(path, renamed_path),
-         {:ok, _output} <- compress_video(renamed_path, output_path) do
+         {:ok, _output} <- compress_video(renamed_path, output_path),
+         true <- FileSizeChecker.get_file_size(output_path) < FileSizeChecker.get_file_size(renamed_path) do
```

Comment:
What happens if you get a `{:error, :cannot_stat_file}` from one of these functions? Elixir will still happily try and compare them.

Reply:
That's the beauty: if one of them is wrong, it will use the one that is correct